docs/reference/esql/processing-commands/keep.asciidoc

```diff
@@ -10,6 +10,7 @@ KEEP columns
 ----
```

Comment:
Very out-of-scope remark: In the commands page, we sometimes spell out comma-seperated lists of columns in the syntax (e.g. in sort and stats: `STATS [column1 =] expression1[, ..., [columnN =] expressionN] ...`), sometimes we just write `columns` like here.